WordPress/src/androidTest/java/org/wordpress/android/ui/screenshots/WPScreenshotTest.java

```diff
@@ -131,60 +126,173 @@ private void screenshotPostWithName(String name, String screenshotName, boolean
         waitForElementToBeDisplayed(R.id.editor_activity);
 
         // Wait for the editor to load all images
-        idleFor(5000);
+        idleFor(7000);
 
         if (hideKeyboard) {
             Espresso.closeSoftKeyboard();
         }
 
+        setNightMode(true);
+        idleFor(2000);
+        setNightMode(isNightMode);
+        idleFor(2000);
```

Comment:
I've cleaned this up as well

Reply:
So to note what was originally happening here and the attempted workaround, some images were taking REALLY(>10s) long to load, however would load after immediately switching the theme. This was an attempt to always switch the theme. I do believe this was however a consequence of running this locally. I think CI performance should be bit better and this would now be unnecessary, I've cleaned it up a bit.